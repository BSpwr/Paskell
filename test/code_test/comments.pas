program CommentTest;
begin

   (* 
      writeln('Not print1!');
      writeln('Not print2!'); 
   *)

    writeln('Print1!');

   // writeln('Not print3!');

    writeln('Print2!');
   
end.