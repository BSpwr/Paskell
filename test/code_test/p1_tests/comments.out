Print1!
Print2!
